test/unit/express-tooling.ts

```diff
@@ -44,10 +45,8 @@ describe('express tooling', () => {
           scope: ['uid', 'test']
         };
         const requiredScopes = ['uid', 'test', 'additional'];
-        let called = false;
-        const next = () => {
-          called = true;
-        };
+
+        const next = () => { return {}; };
```

Comment:
Good catch but would make something like: `const next = () => undefined;` return an object here is a bit confusing

Reply:
true - was just fixing the unused variable used before - TS 2.6.2 is a bit picky on this